lib/rust/flexer/src/automata/state.rs

```diff
@@ -1,70 +1,33 @@
 //! This module exports State implementation for Nondeterministic Finite Automata.
 
-use crate::automata::alphabet::Alphabet;
+use crate::automata::alphabet_segmentation::AlphabetSegmentation;
 use crate::automata::state;
+use crate::automata::symbol::Symbol;
 
 use std::ops::RangeInclusive;
```

Comment:
jsut import prelude instead of suchthings - this is very, very standard thing, just should be in scope, like `default`.